packages/tx/test/eip1559.spec.ts

```diff
@@ -118,10 +118,24 @@ describe('[FeeMarket1559Tx]', () => {
     }
   })
 
+  it('sign() - should default to hedged signatures', () => {
+    const privKey = hexToBytes(eip1559Data[0].privateKey)
+    const txn = createFeeMarket1559Tx({}, { common })
+    // Verify 1000 signatures to ensure these have unique hashes (hedged signatures test)
+    const hashSet = new Set<string>()
```

Comment:
It's a set of hashes, not a `HashSet` data structure :stuck_out_tongue_winking_eye: